cmd/create.go

```diff
@@ -315,6 +324,22 @@ func isValidTemplate(client *fn.Client, runtime, template string) bool {
 	return false
 }
 
+// noRuntimeError creates an error stating that the language flag
+// is required, and a verbose list of valid options.
+func noRuntimeError(client *fn.Client) error {
+	b := strings.Builder{}
+	fmt.Fprintf(&b, "Required flag \"language\" not set.\n")
+	fmt.Fprintln(&b, "Available language runtimes are:")
```

Comment:
As I mentioned before, we should really try to offer available runtimes in a prompt and let user select and proceed.

Reply:
I'm not opposed to that, but I think the solution that @lkingland and @Shashankft9 came up with in [this exchange](https://github.com/knative-sandbox/kn-plugin-func/issues/553#issuecomment-966952816) is good.

Reply:
OK, so then I am probably the only one who think that actually offering a runtime and proceed with the selection instead of an error msg is a better from UX (or more user friendly) 🤷‍♂️ I can live with that :)

Reply:
@zroubalik I see your point - your comment in 633 is convincing.

Reply:
so how do we plan to go forward with this? :)
I like what @zroubalik mentions, but I think we should go ahead with the way it is right now, with confirm and non-confirm modes kept separate, and since we are anyway going to come back to this particular issue as described in #553 by @lkingland after the work in `func config`, we can take another stab at this if there is an ask, what do you think?

Reply:
Well I think that _eventually_ we want to have a nice config option, but for now we can go ahead with this if everyone else agrees this is a good step in the right direction.